tests/conftest.py

```diff
@@ -17,10 +17,18 @@
 from devito.ir.iet import Iteration
 from devito.tools import as_tuple
 
+try:
```

Comment:
don't we simply want ``from devito import MPI`` ?

Reply:
in fact, we could rather add a function "mpi_available()" to ``devito/mpi/__init__.py`` that returns true/false depending on whether mpi4py/MPI is available on the system. Otherwise, we would have to repeat this trick in all module needing MPI...

Reply:
No we wouldn't, MPI exists even without mpi4py, and why `import MPI` would work. This one is just checking if mpi4py is installed but that's the only place like it except inside `mpi`